package.json

```diff
@@ -65,34 +66,34 @@
     "ionicons": "2.0.1",
     "leaflet": "1.2.0",
     "nebular-icons": "1.0.9",
-    "ng2-ckeditor": "1.1.13",
+    "ng2-ckeditor": "^1.2.2",
     "ng2-completer": "2.0.8",
     "ng2-smart-table": "1.3.5",
-    "ngx-echarts": "2.0.1",
+    "ngx-echarts": "^4.0.1",
     "normalize.css": "6.0.0",
     "pace-js": "1.0.2",
     "roboto-fontface": "0.8.0",
-    "rxjs": "^6.1.0",
-    "rxjs-compat": "^6.1.0",
+    "rxjs": "6.3.0",
+    "rxjs-compat": "6.3.0",
     "socicon": "3.0.5",
     "tinymce": "4.5.7",
     "typeface-exo": "0.0.22",
     "web-animations-js": "2.2.5",
     "zone.js": "^0.8.26"
   },
   "devDependencies": {
-    "@angular/cli": "6.0.0",
-    "@angular/compiler-cli": "6.0.0",
-    "@angular/language-service": "6.0.0",
+    "@angular-devkit/build-angular": "0.10.2",
```

Comment:
Don't we need `@angular/cli`?
Just tying out suggestion feature, check the code before committing.
```suggestion
    "@angular/cli": "7.0.3",
    "@angular-devkit/build-angular": "0.10.2",
```

Reply:
It was returned in the latest commit)